test/model/content_test.dart

```diff
@@ -521,6 +521,16 @@ class ContentExample {
       blockUnimplemented('more text'),
     ]]),
   ]);
+
+  static const thematicBreaks =  ContentExample(
```

Comment:
nit: there's only one of these, so singular:
```suggestion
  static const thematicBreak =  ContentExample(
```